libs/app-loader/src/index.ts

```diff
@@ -315,6 +315,11 @@ export default class AppLoader {
     if (!loginData?.id) {
       return;
     }
+
+    // initialise browser notifications
+    const sdk = getSDK();
+    await sdk.api.profile.initNotificationsStream();
```

Comment:
I think this method needs to have support to specify if it should use the signer(signature request- auth push sdk) or it should just use the current eth public address(no signature - resume). Also it should treat the case where the currently signed in user does not have an eth address(ex: no supported)

Reply:
In this specific use-case it should always try to resume and not trigger the signature request.